en/bundle.ftl

```diff
@@ -48,3 +48,45 @@ bundle-banner-dashboard-header = Introducing: { -brand-name-relay } + { -brand-n
 bundle-banner-dashboard-body = Upgrade your subscription to get both { -brand-name-relay-premium } Phone Masking + { -brand-name-mozilla-vpn } for { $monthly_price }/month. Upgrade now to get { $savings } off { -brand-name-mozilla-vpn }!
 bundle-banner-dashboard-upgrade-cta = Upgrade now
 
+megabundle-banner-header = Protect your online privacy for { $monthly_price }/month
+megabundle-banner-header-tools = Get 3 tools for 1 unbeatable price
+megabundle-banner-plan-modules-vpn = { -brand-name-vpn }
+-brand-name-monitor = Monitor
```

Comment:
Add all brand names in this file: https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/brands.ftl